internal/cli/cli.go

```diff
@@ -199,6 +201,9 @@ func (c *RootCommand) runE(cmd *cobra.Command, args []string) error {
 	drlr := viper.GetInt32("default-requests-per-unit")
 	rlc := ratelimit.NewRateLimitCalculatorOrDie(drlu, drlr)
 
+	sensorHandler := sadd.NewHandler(nsInformer, rlc)
+	var eventSourceHandler *esadd.Handler
+
 	if viper.GetBool("enable-webhook-controller") {
```

Comment:
nit: it seems this flag has taken on a different meaning e.g. `enable-event-source-controller`

Reply:
No this flag has the same meaning. The bug was from the previous PR that placed the sensorHandler behind the event source controller feature flag when it should not have been.